### PR TITLE
docs: nav updates for assess dashboards topic

### PR DESCRIPTION
**Pull request by imatwawana**
Updates screenshots in Data source insights section of the page for 9.5 nav updates.
Also removes redundant availability notes; first note on the page covers all the other features on the page.

### CODE REVIEW

File: docs/sources/dashboards/assess-dashboard-usage/index.md [hunk 1]
```diff
@@ -74,17 +70,16 @@ Data source insights provides information about how a data source has been used
 
 To find data source insights:
 
-1. Go to the Data source list view.
-1. Click on a data source.
+1. Go to the **Connections**.
```
Comment (chri2547) [inline docs/sources/dashboards/assess-dashboard-usage/index.md:73]: are they clicking Connections?

**Review by chri2547 — APPROVED** [review]
one question

File: docs/sources/dashboards/assess-dashboard-usage/index.md (hunk 1 above)
Comment (imatwawana) [inline docs/sources/dashboards/assess-dashboard-usage/index.md:73]:
```suggestion
1. Click **Connections** in the main navigation.
```

Comment (imatwawana) [reply docs/sources/dashboards/assess-dashboard-usage/index.md:73]: whoops, missed that!

**Pull request merged**